spaceship/invite.py

```diff
@@ -0,0 +1,64 @@
+
+from flask import render_template, url_for
+import logging
+import re
+from typing import List, Optional
+
+from spaceship import email, achievements
+from spaceship.models import Invitation, Team, User
+
+log = logging.getLogger('spaceship.invite')
+
+class Invite:
+  DEFAULT_SUBJECT = "Join my Spaceship Earth crew!"
+  EMAIL_REGEX = re.compile(r"""[^@\s,]+@[^\s,]+""")
+
+  @classmethod
+  def default_message(cls, inviter: User, team: Team):
+    return render_template('invite_crew.html', inviter=inviter, team=team)
+
+  @classmethod
+  def perform(cls, inviter: User, team_id: int, subject: str, message: str, emails: List[str]) -> Optional[str]:
+    team = Team.query.get(team_id)
+    if not team or team not in inviter.teams:
+      return f'You are not authorized to invite crew onto team {team_id}'
```

Comment:
if the user is seeing this error message, we should show team name instead of team_id